commands/ls.go

```diff
@@ -21,8 +21,10 @@ import (
 	"github.com/skarademir/naturalsort"
 )
 
+const DefaultTimeoutDuration = 10
```

Comment:
I don't think it's exported so it probably should be lowercase.  Also, I think indicating it's for `ls` in particular would be good: `lsTimeoutSeconds` ?